arangod/RestServer/LanguageCheckFeature.cpp

```diff
@@ -56,70 +56,113 @@ arangodb::Result readLanguage(arangodb::ArangodServer& server,
       return TRI_ERROR_INTERNAL;
     }
     VPackSlice defaultSlice = content.get("default");
-    if (!defaultSlice.isString()) {
+    VPackSlice icuSlice = content.get("icu-language");
+
+    // both languages are specified in files
+    if (defaultSlice.isString() && icuSlice.isString()) {
+      return TRI_ERROR_INTERNAL;
```

Comment:
Let's  add a log record  - so we could detect the problem in logs

Reply:
fixed